Frameworks/UIKit/UISegment.mm

```diff
@@ -36,53 +36,56 @@ + (instancetype)initialize {
 }
 
 - (instancetype)initWithCoder:(NSCoder*)coder {
-    id result = [super initWithCoder:coder];
-    id info = [[coder decodeObjectForKey:@"UISegmentInfo"] retain];
-    _position = [coder decodeInt32ForKey:@"UISegmentPosition"];
+    if (self = [super initWithCoder:coder]) {
+        id info = [[coder decodeObjectForKey:@"UISegmentInfo"] retain];
+        _position = [coder decodeInt32ForKey:@"UISegmentPosition"];
 
-    CGRect frame;
-    frame = [self bounds];
+        CGRect frame;
```

Comment:
Since you're in this code, would you mind collapsing these 2 lines into one: CGRect frame = [self bounds]